### PR TITLE
formatted doctsring according to documentations

### PR DESCRIPTION
**Pull request by viratkohli1969**
#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/geometry/entity.py [hunk 1]
```diff
@@ -97,17 +97,23 @@ def __cmp__(self, other):
         return (i1 > i2) - (i1 < i2)
 
     def __contains__(self, other):
-        """Subclasses should implement this method for anything more complex than equality."""
+        """Subclasses should implement this method for anything more complex than equality.
```
Comment (sylee957) [inline sympy/geometry/entity.py:100]: I don't think that we need to add the spaces here, especially if it is docs for private or special methods that are not visible.

**Comment by namannimmo10** [comment]
Travis' log:
>AssertionError: File contains trailing whitespace: /home/travis/build/sympy/sympy/sympy/geometry/entity.py, line 539.

As you can see there's an extra whitespace that causes the build to fail. You must remove that first.

**Comment by mohitshah3111999** [comment]
> 
> 
> Travis' log:
> 
> > AssertionError: File contains trailing whitespace: /home/travis/build/sympy/sympy/sympy/geometry/entity.py, line 539.
> 
> As you can see there's an extra whitespace that causes the build to fail. You must remove that first.

can you tell me which white

**Comment by mohitshah3111999** [comment]
> 
> 
> Travis' log:
> 
> > AssertionError: File contains trailing whitespace: /home/travis/build/sympy/sympy/sympy/geometry/entity.py, line 539.
> 
> As you can see there's an extra whitespace that causes the build to fail. You must remove that first.

Thanks for the help.

File: sympy/geometry/entity.py [hunk 2]
```diff
@@ -243,7 +243,9 @@ def _sympy_(self):
 
     @property
     def ambient_dimension(self):
-        """What is the dimension of the space that the object is contained in?"""
+        """What is the dimension of the space that the object is contained in?
+
+        """
```
Comment (oscarbenjamin) [inline sympy/geometry/entity.py:248]: Is there some reason to add blank lines in a short docstring like this?

**Comment by czgdp1807** [comment]
Closing this for now due to inactivity and non-critical changes. Thanks for your contributions. If you want to continue on this, please let us know in the comments, the PR will be re-opened.

**Pull request closed without merging**